scripts/mutect2_wdl/mutect_resources.wdl

```diff
@@ -0,0 +1,142 @@
+# given a gnomAD vcf, produce:
```

Comment:
Just confirming.... This is a supported WDL?  Just want to make sure since it is not in the `unsupported` directory.

Reply:
Yep, that was intentional.